app/src/main/java/com/zulip/android/networking/AsyncGetEvents.java

```diff
@@ -155,6 +161,7 @@ public void run() {
                 } catch (JSONException e) {
                     backoff(e);
                 }
```

Comment:
Without `Thread.sleep` I receive a traceback on my server 
```
AssertionError
2016-06-22 17:00:08,438 INFO     127.0.0.1       GET     429  11ms /api/v1/events (iago@zulip.com via okhttp)
2016-06-22 17:00:08,439 INFO     status=429, data={"msg":"API usage exceeded rate limit, try again in 36.4631140232 secs","result":"error"}
, uid=iago@zulip.com
2016-06-22 17:00:08,441 ERROR    Uncaught exception GET /api/v1/events?queue_id=1466627219%3A4&last_event_id=19& (127.0.0.1)
HTTPRequest(protocol='http', host='localhost:9993', method='GET', uri='/api/v1/events?queue_id=1466627219%3A4&last_event_id=19&', version='HTTP/1.0', remote_ip='127.0.0.1', body='', headers={'Host': 'localhost:9993', 'Client': 'Android', 'Accept-Encoding': 'gzip', 'X-Forwarded-Host': '192.168.100.6:9991', 'Connection': 'close', 'Authorization': 'Basic aWFnb0B6dWxpcC5jb206bVJHRjZ1Nmt4V0RBVHo2NU1VbWJXa3AxUzZrSk9mRG0=', 'User-Agent': 'okhttp/3.2.0'})
Traceback (most recent call last):
  File "/srv/zulip-venv-cache/615b1182e7545b42f90c5ecfcad3168c4ccb6246/zulip-venv/local/lib/python2.7/site-packages/tornado/web.py", line 1042, in _execute
    getattr(self, self.request.method.lower())(*args, **kwargs)
  File "/srv/zulip/zerver/management/commands/runtornado.py", line 184, in get
    self.set_status(response.status_code)
  File "/srv/zulip-venv-cache/615b1182e7545b42f90c5ecfcad3168c4ccb6246/zulip-venv/local/lib/python2.7/site-packages/tornado/web.py", line 245, in set_status
    assert status_code in httplib.responses
 
```

I guess this is because OkHttpClient is running faster than the existing Apache HTTP Clients!
We might have to increase/decrease the sleep time!